nb2kg/nb2kg/handlers.py

```diff
@@ -112,7 +112,7 @@ def __init__(self):
     @gen.coroutine
     def _connect(self, kernel_id):
         ws_url = url_path_join(
-            KG_URL.replace('http', 'ws'), 
+            os.getenv('KG_WS_URL', KG_URL.replace('http', 'ws')),
```

Comment:
The notebook server has a configuration option for specifying the websocket URL already: 

>NotebookApp.websocket_url : Unicode
> Default: ''
> 
> The base URL for websockets, if it differs from the HTTP server (hint: it almost certainly doesn’t).
>
> Should be in the form of an HTTP origin: ws[s]://hostname[:port]

https://jupyter-notebook.readthedocs.io/en/stable/config.html

You should be able to access it via `self.settings['websocket_url']` in the handler class where you need it, instead of inventing a new configuration parameter. (See https://github.com/jupyter/notebook/blob/master/notebook/notebookapp.py#L266 for how that setting is populated.)